compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -171,6 +171,22 @@ static llvm::raw_ostream &operator<<(
   return os;
 }
 
+/// Splits the given `Range` vector and returns the `lbs` and the `ubs` as
+/// separate lists.
+static void getBoundsFromRange(ArrayRef<Range> loopRange,
```

Comment:
Matthias has done some work on the ValueBounds interface, maybe there is something you can reuse ?

Reply:
It is just splitting the `ArrayRef<Range>` into two separate lists of `lbs` and `ubs`.